Bump core UI to v0.39.2

ognyan-kostadinov opened the pull request:
**Component**:

<!-- E.g. 'salt', 'containers', 'kubernetes', 'build', 'tests'... -->

**Context**: 

**Summary**:

**Acceptance criteria**: 


---

<!-- Declare one or more issues to close once this PR gets merged -->

Closes: #ISSUE_NUMBER

<!-- If you want to refer to an issue while not closing it, use:

See: #ISSUE_NUMBER

-->

ognyan-kostadinov committed:
Bumped core-ui version in metalk8s UI

ognyan-kostadinov committed:
Updated core-ui in shellUI

bert-e commented:
# Hello ognyan-kostadinov,



My role is to assist you with the merge of this
pull request. Please type `@bert-e help` to get information
on this process, or consult the [user documentation](
https://eve.devsca.com/github/scality/metalk8s/bert-e/doc/user).





*Status report is not available.*

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* one peer





Peer approvals *must* include at least 1 approval from the following list:

* @thomasdanan

* @gdemonet

* @jbwatenbergscality

* @nicolast

* @lucieleonard

* @teddyandrieux

* @chengyanjin

* @alexandre-allard-scality

ognyan-kostadinov committed:
Regenerated package-lock.json

ognyan-kostadinov committed:
Fixed broken tests

ChengYanJin reviewed (dismissed):
LGTM!

JBWatenbergScality commented on shell-ui/src/initFederation/UIListProvider.js:
```diff
@@ -1,16 +1,15 @@
 import { useQuery } from 'react-query';
-import Loader from '@scality/core-ui/dist/components/loader/Loader.component';
-import ErrorPage500 from '@scality/core-ui/dist/components/error-pages/ErrorPage500.component';
+import { Loader, ErrorPage500 } from '@scality/core-ui';
```
To minimise shell ui bundle size maybe we should stick to only importing loader and erropage components. 
E,g zenko ui is not using vega but if we change the imports here the bundle will contain vega which is a very large library so as long as core ui can't be tree shaked I would advise using direct imports in shell ui

ognyan-kostadinov committed:
Fixed some imports to reduce bundle size and removed traces of babelrc in favour of babel.conf.js

ognyan-kostadinov committed:
Upgraded version of core-ui to v0.39.3

ognyan-kostadinov committed:
Updated version of module-federation to 1.1

JBWatenbergScality committed:
ci: Remove babel.config.js before executing cypress

Cypress seems trying to use babel when it detects babel.config.js file which is problematic because it should not need to transpile the UI code at this stage as it has already been transpiled and is statically serve in the CI.

ognyan-kostadinov commented:
/approve

bert-e commented:
# In the queue



The changeset has received all authorizations and has been added to the
relevant queue(s). The queue(s) will be merged in the target development
branch(es) as soon as builds have passed.

The changeset will be merged in:


* :heavy_check_mark: `development/123.0`



The following branches will **NOT** be impacted:

* `development/2.0`
* `development/2.1`
* `development/2.10`
* `development/2.11`
* `development/2.2`
* `development/2.3`
* `development/2.4`
* `development/2.5`
* `development/2.6`
* `development/2.7`
* `development/2.8`
* `development/2.9`



There is no action required on your side. You will be notified here once
the changeset has been merged. In the unlikely event that the changeset
fails permanently on the queue, a member of the admin team will
contact you to help resolve the matter.

*IMPORTANT*

Please do not attempt to modify this pull request.

* Any commit you add on the source branch will trigger a new cycle after the
  current queue is merged.
* Any commit you add on one of the integration branches will be **lost**.

If you need this pull request to be removed from the queue, please contact a
member of the admin team now.




*The following options are set:* **approve**

bert-e commented:
# 


I have successfully merged the changeset of this pull request
into targetted development branches:


* :heavy_check_mark: `development/123.0`



The following branches have **NOT** changed:

* `development/2.0`
* `development/2.1`
* `development/2.10`
* `development/2.11`
* `development/2.2`
* `development/2.3`
* `development/2.4`
* `development/2.5`
* `development/2.6`
* `development/2.7`
* `development/2.8`
* `development/2.9`



Please check the status of the associated issue ARTESCA-4763.

Goodbye ognyan-kostadinov.
